File: internal/quality/parser/testdata/sample.go
package sample

import (
	"fmt"

	bar "example.com/bar"
)

type Widget struct {
	N int
}

func (w Widget) Hello() string {
	fmt.Println("hi")
	return bar.Greet()
}

func MakeWidget() Widget {
	return Widget{N: 1}
}

File: internal/quality/parser/testdata/sample.ts
import { x } from './x';
import './side-effect';
export { y } from './y';

interface IFoo {
	n: number;
}

type Bar = number;

class Widget {
	hello(): string {
		return greet();
	}
}

function makeWidget(): Widget {
	return new Widget();
}
